### ISSUE TITLE
Logic error in PostgreSQL service status check after moving @database

### ISSUE THREAD

**Issue by samuelamram** [007revad/Synology_app_mover#153]
There is a logic error in the PostgreSQL service status check in the syno_app_mover.sh script. The current implementation incorrectly triggers an error when the service is running, rather than when it fails.

**Current Behavior:**
The error message is displayed and error code is returned when the service is successfully running.

```bash
if systemctl status pgsql-adapter.service >/dev/null; then
    ding
    echo "ERROR Failed to start pgsql service!" |& tee -a "$logfile"
    return 1
fi
```

There's actually a logical issue here - the condition is inverted from what you might expect. In bash, systemctl status returns:

0 (success) if the service is running
non-zero (failure) if the service is not running

**Expected Behavior:**
The error should be triggered when the service fails to start. The condition should be negated:

```bash
if ! systemctl status pgsql-adapter.service >/dev/null; then
    ding
    echo "ERROR Failed to start pgsql service!" |& tee -a "$logfile"
    return 1
else
    echo "Started pgsql service" |& tee -a "$logfile"
fi
```

**Steps to Reproduce:**

Run the script and choose to move database
Observe that error message appears when PostgreSQL service is actually running correctly

**Comment by 007revad** [comment]
I see what happened. When I added support for DSM 6 moving `@database` in the latest version I copied and pasted the new code from the "check if stopped" section to the "check if running" section and missed that the "check if running" logic was reversed. 

Fixed in [v4.2.81](https://github.com/007revad/Synology_app_mover/releases/tag/v4.2.81)

**Comment by samuelamram** [comment]
Thank you for this quick fix!
